packages/dapp-example/src/Status.tsx

```diff
@@ -195,10 +198,12 @@ export function Status() {
           <StyledStatusSection>
             <StyledStatusSectionTitle>
               <a href={"https://uvk35bjjqk.execute-api.us-east-2.amazonaws.com/status"} target={"_blank"}>
-                {!takerx ? "" : takerx?.status ? "✅" : "⚠️⚠️⚠️"} Backup Taker X (1):
+                {!takerx ? "" : takerx?.length ? "✅" : "⚠️⚠️⚠️"} Backup Taker X (2):
```

Comment:
status should be checked and not length, i.e, if any one of the balances <= 0 show warning emoji